src/adapterManager.js

```diff
@@ -575,4 +575,11 @@ adapterManager.callBidderError = function(bidder, error, bidderRequest) {
   tryCallBidderMethod(bidder, 'onBidderError', param);
 };
 
+adapterManager.getAdapterCode = function(bidderCode) {
```

Comment:
this is no longer needed; the name is also confusing as `bid.adapterCode` now means something else. Probably best to remove.